GitUpKit/Views/en.lproj/GIQuickViewController.xib

```diff
@@ -49,15 +50,15 @@
                                                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                                                         <subviews>
                                                             <view id="Bpc-ZA-3gm" customClass="GIFlippedView">
-                                                                <rect key="frame" x="0.0" y="0.0" width="300" height="100"/>
+                                                                <rect key="frame" x="0.0" y="0.0" width="300" height="150"/>
```

Comment:
A 50px increase seems a lot, can you provide a screenshot of the updated info area of the XIB?